src/server/server.rs

```diff
@@ -308,29 +308,29 @@ impl<T: RaftStoreRouter, S: StoreAddrResolver> Server<T, S> {
     fn on_writable(&mut self, event_loop: &mut EventLoop<Self>, token: Token) {
         let res = match self.conns.get_mut(&token) {
             None => {
-                warn!("missing conn for token {:?}", token);
+                debug!("missing conn for token {:?}", token);
                 return;
             }
             Some(conn) => conn.on_writable(event_loop),
         };
 
         if let Err(e) = res {
-            warn!("handle write conn err {:?}, remove", e);
+            debug!("handle write conn err {:?}, remove", e);
             self.remove_conn(event_loop, token);
         }
     }
 
     fn write_data(&mut self, event_loop: &mut EventLoop<Self>, token: Token, data: ConnData) {
         let res = match self.conns.get_mut(&token) {
             None => {
-                warn!("missing conn for token {:?}", token);
+                debug!("missing conn for token {:?}", token);
                 return;
             }
             Some(conn) => conn.append_write_buf(event_loop, data),
         };
 
         if let Err(e) = res {
-            warn!("handle write data err {:?}, remove", e);
+            debug!("handle write data err {:?}, remove", e);
```

Comment:
what I said before that the store connection can't be closed arbitrarily, so only use warn here to tell us that store connection is removed.